### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by olegpaliy**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
+            <a href="#apple" class="link is-active">APPLE</a>
+          </li>
+          <li class="nav-link">
+            <a href="#samsung" class="link">SAMSUNG</a>
+          </li>
+          <li class="nav-link">
+            <a href="#smartphones" class="link">SMARTPHONES</a>
+          </li>
+          <li class="nav-link">
+            <a href="#laptops" class="link"
```
Comment (YegorKorenkov) [inline src/index.html:35]: Please re-read point 13 in checklist
image

File: src/style.css [hunk 2]
```diff
@@ -1 +1,70 @@
 /* Styles go here */
+
+* {
```
Comment (YegorKorenkov) [inline src/style.css:3]: point 9 in checklist:
image

File: src/style.css [hunk 3]
```diff
@@ -1 +1,70 @@
...
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav-links {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  font-style: normal;
```
Comment (YegorKorenkov) [inline src/style.css:28]: Why you add this rule if it doesn't affect anything?

File: src/style.css [hunk 4]
```diff
@@ -1 +1,70 @@
...
+  line-height: 14px;
+  text-transform: uppercase;
+  color: black;
+}
+
+.nav-link {
+  margin-right: 20px;
+}
+
+.nav-link:last-child {
```
Comment (YegorKorenkov) [inline src/style.css:40]: You can use `.nav-link:not(:last-child)` selector to not apply margin-right to the last element, it shorthen a bit your code

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]
Good job! Please fix minor mistakes

File: src/style.css [hunk 5]
```diff
@@ -1 +1,70 @@
...
+* {
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.logo {
+  margin-top: 5px;
```
[the remark this answers is not in the document]
Comment (olegpaliy) [reply src/style.css]: ok, thank you, changes are pushed,

File: src/style.css (hunk 3 above)
Comment (olegpaliy) [reply src/style.css]: ok, thank you, changes are pushed,

File: src/style.css (hunk 4 above)
Comment (olegpaliy) [reply src/style.css]: ok, thank you, changes are pushed,

File: readme.md [hunk 6]
```diff
@@ -1,18 +1,22 @@
 # Moyo header
+
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+
+- [DEMO LINK](https://olegpaliy.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://olegpaliy.github.io/layout_moyo-header/report/html_report/)
```
Comment (And678) [inline readme.md:6]: Don't forget to add these links to PR description

**Review by And678 — APPROVED** [review]
Nice work fixing those! Approve :heavy_check_mark: , but check my comment below

**Pull request closed without merging**